newspaper-api.yml

```diff
@@ -45,7 +50,7 @@ spec:
     spec:
       containers:
       - name: master
-        image: smarp/newspaper-api
+        image: smarp/newspaper-api:v2.0
```

Comment:
now we will need to manually specify tag, right?

Reply:
no, it's not something new; actually I find it really strange that it wasn't set...

Reply:
right